optimum/exporters/openvino/model_patcher.py

```diff
@@ -1865,6 +1872,68 @@ def __exit__(self, exc_type, exc_value, traceback):
                 layer.self_attn.forward = layer.self_attn._orig_forward
 
 
+# copied from  https://github.com/huggingface/optimum/blob/v1.22.0/optimum/bettertransformer/models/attention.py#L169
```

Comment:
Can you please add a fixed commit in the link so that it will be valid later?

Reply:
```suggestion
# copied from  https://github.com/huggingface/optimum/blob/2112e99122d7f23a1da1a9d263fef64301050ea7/optimum/bettertransformer/models/attention.py#L168
```

Reply:
> v1.22.0 it is tag in the repo https://github.com/huggingface/optimum/releases/tag/v1.22.0 it is already pointed to specific commit. Do you mean what happens if somebody remove this tag?

Ah, I see. I think it's fine then.